### PR TITLE
RevEng - Fix for issue 1934 - improve layout...

### PR DESCRIPTION
**Pull request by lajones**
...of configuration code in OnModelCreating(). See #1934.

### CODE REVIEW

**Comment by bricelam** [comment]
Tests? We seem to have gotten into a very bad habit.

**Comment by lajones** [comment]
@bricelam They're coming. Am working on them now.

**Comment by divega** [comment]
@bricelam there will be tests that will compare the codegen output with a baseline. These changes are being made first just to avoid reworking the tests (at least this first time). Please keep that in mind when reviewing.

File: src/EntityFramework.SqlServer.Design/ReverseEngineering/Templates/SqlServerDbContextTemplate.cshtml [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+                {
+                    lines.Add("." + forMethod + "()." + methodBodyList.First());
+                }
+                else
+                {
+                    lines.Add("." + forMethod + "(" + propertyLambdaIdentifier + " =>");
+                    lines.Add("{");
+                    foreach (var methodBody in methodBodyList)
+                    {
+                        lines.Add(indent + propertyLambdaIdentifier + "." + methodBody + ";");
```
Comment (bricelam) [inline src/EntityFramework.SqlServer.Design/ReverseEngineering/Templates/SqlServerDbContextTemplate.cshtml]: Any reason we're not using the indented string builder?

**Comment by bricelam** [comment]
:shipit:

**Comment by lajones** [comment]
Merged with commit c66035f.

**Pull request closed without merging**